lib/Basics/StringUtils.cpp

```diff
@@ -1963,6 +1963,37 @@ std::string escapeRegexParams(std::string const& in) {
   return out;
 }
 
+std::string formatSize(uint64_t value) {
+  std::string out, label;
+  if (value < 1000) {
+    if (value == 1) {
+      out = "1";
+      label ="byte";
+    } else {
+      out = std::to_string(value);
+      label = "bytes";
+    }
+  } else if (value < 1000000ULL) {
+    out = std::to_string((double) value / double(1000));
+    label = "KB";
+  } else if (value < 1000000000ULL) {
+    out = std::to_string((double) value / double(1000000));
```

Comment:
```suggestion
    out = std::to_string((double) value / 1e6);
```
If you don't like the scientific notation I would vote for using `double(1'000'000)` instead.